fdbclient/NativeAPI.actor.cpp

```diff
@@ -10099,15 +10096,20 @@ ACTOR Future<Standalone<VectorRef<KeyRangeRef>>> listBlobbifiedRangesActor(Refer
                                                                            KeyRange range,
                                                                            int rangeLimit) {
 	state Database db(cx);
-	state Reference<ReadYourWritesTransaction> tr = makeReference<ReadYourWritesTransaction>(db);
-
-	state Standalone<VectorRef<KeyRangeRef>> blobRanges = wait(getBlobRanges(tr, range, rangeLimit));
+	state Transaction tr(db);
 
-	return blobRanges;
+	loop {
+		try {
+			state Standalone<VectorRef<KeyRangeRef>> blobRanges = wait(getBlobRanges(&tr, range, rangeLimit));
+			return blobRanges;
```

Comment:
This fixes the original comment, but the one potential improvement would be to record your progress before the error. That doesn't have to be in the scope of this PR though
For example, if I am reading [A - Z), and I read up through [A - V) before getting a retryable error, i'd re-read all of [A - V) on the second retry.